### PR TITLE
updated codeowners

### PR DESCRIPTION
**Pull request by AmrutaKawade**
**Task name**: NA

**Description**: Updated CodeOwners

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N
**Attached related issue:** (Y/N) N

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -107,7 +107,7 @@ Tasks/Common/artifacts-common/     @phil-hodgson    @aasim    @satbai
 
 Tasks/Common/AzureRmDeploy-common/   @vincentdass  @SumiranAgg
```
Comment (chshrikh) [inline .github/CODEOWNERS:108]: change this as well

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -107,7 +107,7 @@ Tasks/Common/artifacts-common/     @phil-hodgson    @aasim    @satbai
...
 
-Tasks/Common/Deployment/   @bishal-pdMSFT  @chshrikh
+Tasks/Common/Deployment/   @bishal-pdMSFT  @chshrikh
```
Comment (chshrikh) [inline .github/CODEOWNERS:110]: add Naveen as well here

**Review by chshrikh — APPROVED** [review]
added two comment pls take care and merge

File: .github/CODEOWNERS (hunk 1 above)
Comment (AmrutaKawade) [reply .github/CODEOWNERS]: done

File: .github/CODEOWNERS (hunk 2 above)
Comment (AmrutaKawade) [reply .github/CODEOWNERS]: done

**Pull request merged**